proto/sifnode/dispensation/v1/tx.proto

```diff
@@ -9,6 +9,8 @@ option go_package = "github.com/Sifchain/sifnode/x/dispensation/types";
 service Msg {
   rpc CreateDistribution(MsgCreateDistribution)
       returns (MsgCreateDistributionResponse);
+  rpc CreateClaim(MsgCreateUserClaim)
```

Comment:
Can we keep consistency here, i.e CreateClaim(MsgCreateClaim)

Reply:
Ill change this to CreateUserClaim(MsgCreateUserClaim) . It will maintian consistency with the rest of the module